tools-local/tasks/installer.tasks/CreateFrameworkListFile.cs

```diff
@@ -186,6 +186,11 @@ public override bool Execute()
                     }
                 }
 
+                if (f.IsNative && !ShouldIncludeInSingleFileHost(f.Filename))
```

Comment:
This should be done similarly to `ReferencedByDefault`, not hard-coded into the build task itself, so that putting it into Arcade doesn't prevent quickly changing it in dotnet/runtime. (Also makes the data easier to find.)

Reply:
When it was supposed to be entirely in runtime, it could be more convenient to keep the logic in one place. 
If this needs to go into arcade, I guess adding it as metadata on Files could make sense.